### PR TITLE
Fix pid unlink error, refactor a bit

### LINKED ISSUE #22: A JavaScript error occured in the browser process
This happened when I shut down a process:

screen shot 2015-05-09 at 9 29 02 pm

```
Uncaught Exception:
Error: ENOENT: no such file or directory, unlink '/Users/z/Library/Application Support/Monu/data/pids/dot-clipboard.pid'
    at Error (native)
    at Object.fs.unlinkSync (fs.js:939:18)
    at Process.removePidfiles (/Applications/Monu.app/Contents/Resources/app/node_modules/mongroup/lib/process.js:101:6)
    at null._onTimeout (/Applications/Monu.app/Contents/Resources/app/node_modules/mongroup/lib/process.js:197:10)
    at Timer.listOnTimeout (timers.js:88:15)
```

Despite the error, the process did stop as intended.

### PR DESCRIPTION
**Pull request by marcbachmann**
This fixes #22
I've also refactored some overloaded methods and moved some code around.
It should give a better foundation for further updates.

### CODE REVIEW

File: configure.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
-var ipc = require('ipc')
 var Ractive = require('ractive')
 var page = require('page')
 var fs = require('fs')
+var Client = require('electron-rpc/client')
+var client = new Client()
```
Comment (max-mapper) [inline configure.js:5]: whoa didnt know you could do that, is it documented? (the `electron-rpc` module). im assuming its the same thing as ipc?

Comment (max-mapper) [inline configure.js:5]: oh wait lol, i should read the full PR first, didn't realize it was a third party module

Comment (marcbachmann) [inline configure.js:5]: :smile:

**Comment by max-mapper** [comment]
cool, LGTM

**Comment by marcbachmann** [comment]
Will do another commit on that. I'll move all the methods outside of the menu.on('ready') event.

File: configure.js (hunk 1 above)
Comment (marcbachmann) [inline configure.js:5]: If we want to use more features, we can also start a whole express server on a random port. But currently there's no need for it.

**Pull request merged**